opencog/timeoctomap/TimeOctomap.cc

```diff
@@ -29,67 +29,71 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-//TimeSpaceAtom.cpp
-#include "TimeSpaceAtom.h"
+//TimeOctomap.cpp
+#include "TimeOctomap.h"
 //#include "octomap/OcTreeKey.h"
 //#include <assert.h>
 #include "opencog/util/oc_assert.h"
-TimeSpaceAtom::TimeSpaceAtom(unsigned int num_time_units,
-                             vector<double>map_res_meters):
-    time_circle(num_time_units), created_once(false)
+
+TimeOctomap::TimeOctomap(unsigned int num_time_units,
+                         double map_res_meters,
+                         duration_c time_resolution):
+    time_circle(num_time_units),map_res(map_res_meters),time_res(time_resolution),created_once(false)
```

Comment:
I'm an emacs. I looked into tab vs space autodetection a few months ago for emacs but it didn't work well (though I'm sure it wouldn't be hard to fix). Don't know if there is something like that for vim.